open-sauced/app
# chore: add eslint jsx a11y package

## Description

This PR adds the [eslint jsx a11y plugin](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) to the project.

## Related Tickets & Documents

Fixes #2264 

## Mobile & Desktop Screenshots/Recordings


N/A


## Steps to QA


1. Run `npx eslint .` at root of project
2. See the accessibility issues show up as warnings instead of errors


## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?

![A GIF of a man in a black suit coat and a gray inner shirt is pulling his sleeves then flexing his collar](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExNG5sdmF4aWp6eTg0bWJvODhpNWdlamVvZ3U1NzRxZzJuenlxbmp6ZCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/65Bfas6vezOyRUyRn7/giphy.webp)
